report-hazard/index.html

```diff
@@ -0,0 +1,134 @@
+                <input type="radio" name="category" value="Infrastructure">Infrastructure<br>
+                <input type="radio" name="category" value="Campsite">Campsite<br>
+                <a href="#step1">Back</a>
+                <a href="#step3">Continue</a>
+            </div>
+            <div class="page" id="step3">
+                <h1>3. What type of wild animal did you spot?</h1>
+                <input type="radio" name="type" value="Bear">Bear<br>
+                <input type="radio" name="type" value="Coyote">Coyote<br>
+                <input type="radio" name="type" value="Coguar">Coguar<br>
+                <input type="radio" name="type" value="Elk">Elk<br>
+                <input type="radio" name="type" value="Raccoon">Raccoon<br>
+                <a href="#step2">Back</a>
+                <a href="#step4">Continue</a>
+            </div>
+
+            <div class="page" id="step4">
+                <h1>4. Would you like to mention any further details?</h1>
+                <input type="text" id="details">
+                <br>
+                <a href="#step3">Back</a>
+                <a href="#step5">Continue</a>
+            </div>
+
+            <div class="page" id="step5">
+                <h1>5. Please upload photos to help others see the reported hazard visually. </h1>
+                <p>Upload pictures</p>
+                <input type="text" id="picture">
+                <br>
+                <a href="#step4">Back</a>
+                <a href="#step6" onClick="myFu()">Continue</a>
+                <!-- <input type="submit" class="submit" value="Continue"/> -->
+
+                <!-- <input type="submit" class="submit" value="Continue" onClick="myFu()"/>
+                <script>
+                    function myFu(){
+                        window.location.href="#step6"
+                    }
+                </script> -->
+
+                <!-- <button id="button5" class="button" onClick="myFu()">Continue</button> -->
+
+                <!-- <script>
+                    function myFu(){
+                        const locationInput = form.querySelector("#name")
+    console.log(locationInput.value)
+                        window.location.href="#step6"
+                    }
+                </script> -->
+
+            </div>
+
+            <div class="page" id="step6">
+                <h1>6. Review the report before submitting.</h1>
+                <!-- <h2>Location:</h2>
+                <h2>Type of hazard:</h2>
+                <h2>Hazard:</h2>
+                <h2>Additional details:</h2>
+                <h2>Photos:</h2> -->
+                <br>
+                <div id="review">
+
+                </div>
+                <br>
+                <a href="#step5">Back</a>
+                <!-- <a href="#step7">Submit Report</a> -->
+                <input type="submit" class="submit" onClick="confiFu()">
+                <script>
```

Comment:
move all javascript to the script.js file

Reply:
commit 5706ba965f5232c320ae3d7a33164bd8b920b306

Reply:
commit 17986c3376a6887fb6a3c778bc535db147fdd45c